dix/dispatch.c

```diff
@@ -1100,13 +1104,10 @@ ProcGetAtomName(ClientPtr client)
     };
```

Comment:
We should remove unnecessary initializations as usual.

Reply:
indeed.
thx.